.github/workflows/build.yml

```diff
@@ -23,3 +23,33 @@ jobs:
     - name: build_and_test
       run: ./build_and_test.sh clean
 
+  build-alpine-latest:
+
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+    - name: Setup latest Alpine Linux
+      uses: jirutka/setup-alpine@v1
```

Comment:
I deliberately avoid these plugins, to avoid vendor lock-in, in case I have to migrate to GitLab or some other similar service in the future. We depend soley on free infrastructure, because we have zero funding.

I think you have hard part done. Could we instead wrap in a podman or docker container directly and run this in one of the shell scripts?

The less .yml and plugins, the better!

Reply:
An example is like here https://github.com/ericcurtin/twincam/blob/main/tests/run-all.sh